app/policies/casa_case_policy.rb

```diff
@@ -17,7 +17,7 @@ def initialize(user, scope)
     def resolve
       case @user
       when CasaAdmin, Supervisor
-        scope
+        scope.where(casa_org: @user.casa_org)
```

Comment:
Would it make sense given the context to move it into a `scope` in the model?

Reply:
yes please :)